bundles/org.openhab.binding.orbitbhyve/src/main/resources/OH-INF/thing/channels.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="orbitbhyve"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<!-- Mode Channel Type -->
+	<channel-type id="mode">
+		<item-type>String</item-type>
+		<label>Irrigation Mode</label>
+		<description>Channel representing mode of Orbit B-Hyve Device (auto/manual)</description>
+		<state readOnly="true"></state>
+	</channel-type>
+	<channel-type id="next_start">
+		<item-type>DateTime</item-type>
+		<label>Next Watering Start Time</label>
+		<description>Channel representing start time of the next watering</description>
+		<state readOnly="true"></state>
+	</channel-type>
+	<channel-type id="rain_delay">
+		<item-type>Integer</item-type>
+		<label>Rain delay</label>
```

Comment:
Words in labels should be capitalized (except prepositions and so on). See https://www.openhab.org/docs/developer/bindings/thing-xml.html#formatting-labels-and-descriptions

Reply:
Fixed label capitalisation (more occurences)